compile/transform.ts

```diff
@@ -1,5 +1,5 @@
 import { EditorState, Mould, Component } from '../app/types'
-import * as m from '../index'
+import * as m from '../mould'
```

Comment:
There's no reason to import from `index`, because Mould components were moved to `mould` (please see https://github.com/mouldjs/mould/pull/79#discussion_r440933731)